test/DriverFactoryTest.php

```diff
@@ -42,6 +42,6 @@ public function testItSupportsGlobalBasenameOptionOnFileDrivers()
         $factory = new DriverFactory();
 
         $driver = $factory($container->reveal());
-        $this->assertSame($globalBasename, $driver->getGlobalBasename());
+        static::assertSame($globalBasename, $driver->getGlobalBasename());
```

Comment:
For consistency with the rest of the project I wouldn't call `assertSame()` statically.